lib/compat.php

```diff
@@ -238,12 +238,17 @@ function gutenberg_user_settings_data_persistence_inline_script() {
 
 	$persistence_script = <<<JS
 ( function() {
+	window.locallyPersistentValues = {};
```

Comment:
There is no need to pollute the global scope, you can define a local variable enclosed in the function.

```suggestion
	const locallyPersistentValues = {};
```

Reply:
Already fixed :+1: